astropy/modeling/tests/test_models.py

```diff
@@ -115,7 +115,7 @@ def test_inconsistent_input_shapes():
     y = x.copy()
     # check scalar input broadcasting works
     assert np.abs(g(x, 0) - g(x, 0 * x)).sum() == 0
-    # but not array broadcasting
+    # and that array broadcasting works
```

Comment:
But the updated sentence is the complete opposite of the old one. 😆 

Has it always been wrong or is this some follow-up fix of a recent PR?

Reply:
The latter: this was fixed in #11548, but the comment was never updated. Not a particularly recent PR, given that its was from 2021.